app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
+end
```

Comment:
Add validations, you have null validation in the migration, review this https://guides.rubyonrails.org/active_record_validations.html

Reply:
I added the validations when I did the book model test (PR #7)